pkg/reconciler/instances/istio/reset/data/gatherer_test.go

```diff
@@ -54,6 +54,7 @@ func Test_Gatherer_GetPodsWithDifferentImage(t *testing.T) {
 	podWithDifferentImageSuffixTerminating := fixPodWith("application", "kyma", "istio/proxyv2:1.10.2", "Terminating")
 	podWithDifferentImageSuffixPending := fixPodWith("application", "kyma", "istio/proxyv2:1.10.2", "Pending")
 	podWithDifferentImagePrefix := fixPodWith("application", "kyma", "istio/weirdimage:1.10.2", "Running")
+	podWithSoloImagePrefix := fixPodWith("istio", "custom", "istio-69e5957db6d2/proxyv2:1.10.2", "Running")
```

Comment:
Two things here:
1. we should not use `Solo` keyword here as we should not rely on or even mention any external dependency,
2. do not be that specific with Istio image hash - as you can see from above it would be better to use sum buzz wording such as `custom` or `weird` image naming.

Reply:
logic changed